test/e2e/Common.t.sol

```diff
@@ -58,6 +58,13 @@ contract DeployForTest is TestParams, Deploy {
     collateralTypes.push('TKN-C');
     collateralTypes.push('TKN-8D');
   }
+
+  function setupPostEnvironment() public virtual override updateParams {
+    for (uint256 i = 0; i < collateralTypes.length; i++) {
+      bytes32 _cType = collateralTypes[i];
+      taxCollector.taxSingle(_cType);
+    }
+  }
```

Comment:
unnecessary changes?

Reply:
I think it makes the for-test deployment routine tidier.